verilog/analysis/verilog_linter.h

```diff
@@ -34,19 +35,44 @@
 
 namespace verilog {
 
+// Base class of LintRuleStatus handlers
```

Comment:
Here is the place to explain what this is used for, something that explains that the lint reports coming back are passed to an instance of this.

Reply:
Done